### ISSUE TITLE
How to configure .sprite-item?

### ISSUE THREAD

**Issue by ghost** [CurimuChizu/CC-Themes#2]
If I wanted to make my own custom sprites, how would I go by doing so?

Thanks

**Comment by CurimuChizu** [comment]
@bamtan There's a sprite sheet generator on making your own sprite sheet, but the way I did it was by Photoshop, because I couldn't figure out how to use the it, just copy and paste the sprite image and resize it to the current ones.

**Comment by ghost** [comment]
[Something like this? ](https://draeton.github.io/stitches/) What were the dimensions for the sprites?

Also, if I just wanted that sprite feature only, would I remove everything else and leave 
`/* Emoji */`
`.channel-textarea-emoji>.sprite-item,`
`.channel-textarea-emoji.popout-open .sprite-item {`
`background-image:url(https://a.pomf.cat/iyqmjn.png)!important;`
`}`

thanks again !

**Comment by CurimuChizu** [comment]
@bamtan Yeah, that one. Also if you want the sprite emoji code, then yeah you only need that part. 

Anytime. If you have anything else, I'll gladly help if I can.
